api/src/pcapi/alembic/versions/20230602T090227_54974ef1d55a_add_eligibility_type_to_fraud_reviews.py

```diff
@@ -0,0 +1,23 @@
+"""FIXME: the comment below appears in the output of `alembic history`.
+Remove this FIXME and make the comment below easily readable.
+
+add_eligibility_type_to_fraud_reviews
+"""
```

Comment:
N'oublie pas ce FIXME
Qque chose comme `Add eligibility column to BeneficiaryFraudReview table`

Reply:
Et voilà :)